src/test-utils.js

```diff
@@ -1,6 +1,6 @@
-/* eslint-disable import/prefer-default-export */
-
-export const executeThunk = async (thunk, dispatch, getState) => {
+const executeThunk = async (thunk, dispatch, getState) => {
   await thunk(dispatch, getState);
   await new Promise(setImmediate);
 };
+
+export default executeThunk;
```

Comment:
add new line after export statement

Reply:
All the files have a final newline already, it just doesn't show in the changes view.